cmd/serve.go

```diff
@@ -179,18 +181,21 @@ func requireFlag(flagName string, flagValue string) {
 }
 
 func serve() error {
-	keypairFilename := "signing"
-	if err := token.GenerateKeypair(keypairFilename); err != nil {
-		glog.Errorf("Error generating key pair: %v", err)
+	if !token.KeypairExists(keypairDir) && genKeypair {
```

Comment:
If genKeypair is true, I believe it should always generate the keypair. Otherwise you could have a hard to detect startup race in shared volume problem.

Reply:
ok, i will make this change

Reply:
done